docs/DeveloperGuide.md

```diff
@@ -158,37 +158,37 @@ This section describes some noteworthy details on how certain features are imple
 
 #### Proposed Implementation
 
-The proposed undo/redo mechanism is facilitated by `VersionedAddressBook`. It extends `AddressBook` with an undo/redo history, stored internally as an `addressBookStateList` and `currentStatePointer`. Additionally, it implements the following operations:
+The proposed undo/redo mechanism is facilitated by `VersionedSalesNote`. It extends `SalesNote` with an undo/redo history, stored internally as an `SalesNoteStateList` and `currentStatePointer`. Additionally, it implements the following operations:
```

Comment:
salesNoteStateList might be more appropriate here, since if we follow their structure exactly, this is the name of a field and not a class in the implementation I believe